### PR TITLE
Add output delay

### LINKED ISSUE #31: Add printDelay
I was having a requirement to add printDelay as many charts are involved in my url which takes time to load.

### PR DESCRIPTION
**Pull request by chrisocast**
Closes #31

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -86,18 +87,21 @@ function render (indexUrl, output) {
   }
 
   win.webContents.on('did-finish-load', function () {
-    win.webContents.printToPDF(opts, function (err, data) {
-      if (err) {
-        console.error(err)
-      }
 
-      fs.writeFile(path.resolve(output), data, function (err) {
+    setTimeout(function(){
```
Comment (fraserxu) [inline index.js]: Please add a space to `function ()` so it pass [standard](https://github.com/feross/standard) code check.

**Comment by cgwyllie** [comment]
@chrisocast @fraserxu any movement on this/could I help in any way?

We recently discovered this tool and it's working well, but we also needed to add some delay option to allow rendering SPAs. Would prefer to avoid creating a fork if there's a PR waiting.

Thanks

**Comment by fraserxu** [comment]
I'm OK to get this merged after the test is pass. @chrisocast Do you have some time to have a quick fix?

**Comment by codecounselor** [comment]
Closing this because the functionality exists in master

**Pull request closed without merging**